hack/install.sh

```diff
@@ -119,6 +129,24 @@ do_install() {
 	esac
 
 	if command_exists docker; then
+		version="$(docker -v | awk '{ print $3 }' | sed 's/\,//g')"
```

Comment:
(but to be clear, I am still firmly in the "YOLO users really only do
live once" camp WRT this script)